.babelrc

```diff
@@ -1,20 +1,25 @@
 {
-  "stage": 0,
+  "presets": [
```

Comment:
I reckon we can follow the presets specified [here](https://github.com/gaearon/react-transform-boilerplate/blob/master/.babelrc). No issue with having stage-0. But we could look into using `react-hmre`.